core/src/main/resources/struts-default.xml

```diff
@@ -372,6 +374,15 @@
                  associated with requests that should not be validated.
                  -->
             <interceptor-stack name="defaultStack">
+                <interceptor-ref name="coepInterceptor">
+                    <param name="enforcingMode">true</param>
+                    <param name="reportUri">/coep-reports</param>
+                    <param name="disabled">false</param>
+                </interceptor-ref>
+                <interceptor-ref name="coopInterceptor">
+                    <param name="exemptedPaths"></param>
+                    <param name="mode"></param>
```

Comment:
we should probably have a default value here for the `mode` just like the `coepInterceptor`